tests/locale/keymap_or_locale.pm

```diff
@@ -76,20 +87,15 @@ sub verify_default_keymap_x11 {
 }
 
 sub run {
-    # uncomment in case of different keyboard than us is used during installation ( feature not ready yet )
-    # my $expected   = get_var('INSTALL_KEYBOARD_LAYOUT','us');
-    my $expected       = 'us';
-    my %keystroke_list = (
-        us => '`1234567890-=~!@#$%^&*()_+',
-        fr => '²&é"(-è_çà)=~1234567890°+',
-        de => '1234567890ß°!"§$%&/()=?',
-        cz => ';+ěščřžýáíé=1234567890%'
-    );
-    my $keystrokes = $keystroke_list{$expected};
+    my $expected = get_var('INSTALL_KEYBOARD_LAYOUT', 'us');
+    # Feature of switching keyboard during installation is not ready yet,
+    # so if another language is used it needs to be verfied that the needle represents properly
+    # characters on that language.
+    my $keystrokes = '`1234567890-=!@6~!@#$%^&*()_+';
 
     if (check_var('DESKTOP', 'textmode')) {
         assert_screen([qw(linux-login cleared-console)]);
-        verify_default_keymap_textmode($keystrokes, "${expected}_keymap");
+        return if verify_default_keymap_textmode($keystrokes, "${expected}_keymap");
```

Comment:
I'm not sure if this is clear, this condition is weird to me. I would split the method into two and make it explicit that if it's `us` we don't run further checks.

Reply:
I created a function for that to make it more clear. VR updated.